packages/admin/cms-admin/src/generator/future/generateGrid/generateGridToolbar.ts

```diff
@@ -0,0 +1,79 @@
+import { camelCaseToHumanReadable } from "../utils/camelCaseToHumanReadable";
+import { getFormattedMessageNode } from "../utils/intl";
+
+type Options = {
+    renderToolbar: boolean;
+    gqlTypePlural: string;
+    forwardToolbarAction: boolean | undefined;
+    hasSearch: boolean;
+    hasFilter: boolean;
+    allowAdding: boolean;
+    instanceGqlType: string;
+    gqlType: string;
+};
+
+export const generateGridToolbar = ({
+    renderToolbar,
+    gqlTypePlural,
+    forwardToolbarAction,
+    hasSearch,
+    hasFilter,
+    allowAdding,
+    instanceGqlType,
+    gqlType,
+}: Options) => {
+    if (!renderToolbar) {
+        return "";
+    }
```

Comment:
Wouldn't it be better to make this check outside of the function? Then the argument wouldn't be needed.

(Applies for all functions in this file)

Reply:
Done in df347d0b79dd8e03d057f44dd9dc01ade6a4b7bb